src/components/MenuCard/menucard.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  margin-left: 12px;
+  margin-right: 12px;
+  border-radius: 0px;
+  padding: 20px;
+  padding-left: 28px;
+  padding-right: 12px;
+  margin-bottom: 20px;
+  box-shadow: 0px 0px 2px 0px rgba(0, 0, 0, 0.1);
+}
+
+.cardButton {
+  margin-left: 82%;
```

Comment:
Este tipo de propriedade é bem perigosa porque pode quebrar com qualquer variação do viewport ou do tamanho dos elementos. O mesmo vale para os usos aqui de `top`, `bottom` e especialmente `float`. Minha sugestão seria usar dos utilitários que já existem em stylesheets/utilities/, pois nos casos triviais estes devem ser suficiente. No componente poderia ficar organizado assim:
```
<Card className="flex items-center justify-between">
  <div class="flex space-between-inline-m">
    <div class="iconContainer">
      <Icon 
        className="center-vertically" 
        name={icon} 
        height={30} 
        width={30} 
        color={iconColor} 
      />
    </div>
    <div>
      <h3 class="space-stack-s">Simular Prova</h3>
      <p>Selecione e execute uma prova dos anos anteriores</p>
    </div>
  </div>
</Card>
```
Recursos relevantes: [Basic concepts of flexbox](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox) e [Space in design systems](https://medium.com/eightshapes-llc/space-in-design-systems-188bcbae0d62).